.gitignore

```diff
@@ -61,3 +61,5 @@ typings/
 .next
 
 build
+
+*.env
```

Comment:
It's good to avoid checking in the secrets here of course, but it also includes e.g. the scopes which are not secret and would be helpful for anyone else... Maybe two separate files for secrets vs. auth config settings such as scopes that are constant for anyone who wants this dashboard...?

Reply:
Done - there's now a `config.env` and a `secrets.example.env` both checked in